crates/egui_demo_app/src/backend_panel.rs

```diff
@@ -296,6 +296,20 @@ fn integration_ui(ui: &mut egui::Ui, _frame: &mut eframe::Frame) {
                 }
             }
 
+            if ui
+                .button("🖵 Desktop Size")
+                .on_hover_text("Resize the window to be 720p.")
+                .clicked()
+            {
+                let size = egui::vec2(1280.0, 720.0);
+
+                ui.ctx()
+                    .send_viewport_cmd(egui::ViewportCommand::InnerSize(size));
+                ui.ctx()
+                    .send_viewport_cmd(egui::ViewportCommand::Fullscreen(false));
+                ui.close_menu();
+            }
```

Comment:
I vote making the sizing options a drop-down menu so it doesn't use as much space, i.e. `Resize…` -> menu with "Phone sized" and "720p"